config.json

```diff
@@ -12,6 +12,11 @@
       ]
     },
     {
+      "slug": "two-fer",
+      "difficulty": 1,
+      "topics": []
```

Comment:
This topics entry should be formatting like the others (with a blank line between the brackets).